libvips/foreign/heifsave.c

```diff
@@ -363,6 +368,17 @@ vips_foreign_save_heif_build( VipsObject *object )
 		return( -1 );
 	}
 
+	chroma = heif->subsample_mode == VIPS_FOREIGN_SUBSAMPLE_OFF ||
+		( heif->subsample_mode == VIPS_FOREIGN_SUBSAMPLE_AUTO &&
+			heif->Q > 90 ) ? "444" : "420";
```

Comment:
Have used same Q>90 condition as JPEG to prevent subsampling, easy to change this if required.

Reply:
I think this is a good choice!